### PR TITLE
EPMRPP-93205 || Wrong behaviour when clicking on All organizations bu…

### PR DESCRIPTION
**Pull request by BlazarQSO**
…tton in the sidebar

## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/layouts/organizationsControl/organizationsControl.jsx [hunk 1]
```diff
@@ -45,12 +45,18 @@ export const OrganizationsControl = ({
             'instance-level': isExtendedNav,
           })}
         >
-          {!isExtendedNav ? <i className={cx('arrow-icon')}>{Parser(ArrowLeftIcon)}</i> : null}
-          <Link to={link} className={cx('organization-btn')} onClick={closeSidebar}>
-            <div className={cx('top-title')}>{titles.topTitle}</div>
-          </Link>
+          {isExtendedNav ? (
```
Comment (AmsterGet) [inline app/src/layouts/organizationsControl/organizationsControl.jsx:48]: Could you please reverse the `isExtendedNav` boolean value come from upper components?
I mean that when it is `isExtendedNav=true`, the content should be **extended** with smth, but right now with this option the content is shorter than without it.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/layouts/organizationsControl/organizationsControl.jsx (hunk 1 above)
Comment (BlazarQSO) [reply app/src/layouts/organizationsControl/organizationsControl.jsx:48]: Done

Comment (BlazarQSO) [reply app/src/layouts/organizationsControl/organizationsControl.jsx:48]: Done

File: app/src/layouts/organizationsControl/organizationsControl.jsx [hunk 2]
```diff
@@ -42,15 +42,21 @@ export const OrganizationsControl = ({
       <div>
         <div
           className={cx('organization-btn-wrapper', {
-            'instance-level': isExtendedNav,
+            'instance-level': !isExtendedNav,
```
Comment (AmsterGet) [inline app/src/layouts/organizationsControl/organizationsControl.jsx:45]: Let's rename the class to `not-extended` or `plan` or smth like that.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/layouts/organizationsControl/organizationsControl.jsx (hunk 2 above)
Comment (BlazarQSO) [reply app/src/layouts/organizationsControl/organizationsControl.jsx]: Done

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**